contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -95,11 +92,11 @@ namespace eosiosystem {
       const auto usecs_since_last_fill = (ct - _gstate.last_pervote_bucket_fill).count();
 
       if( usecs_since_last_fill > 0 && _gstate.last_pervote_bucket_fill > time_point() ) {
-         auto new_tokens = static_cast<int64_t>( (continuous_rate * double(token_supply.amount) * double(usecs_since_last_fill)) / double(useconds_per_year) );
+         auto new_tokens = static_cast<int64_t>( (_gstate4.continuous_rate * double(token_supply.amount) * double(usecs_since_last_fill)) / double(useconds_per_year) );
 
-         auto to_producers     = new_tokens / inflation_pay_factor;
+         auto to_producers     = new_tokens / _gstate4.inflation_pay_factor;
          auto to_savings       = new_tokens - to_producers;
-         auto to_per_block_pay = to_producers / votepay_factor;
+         auto to_per_block_pay = to_producers / _gstate4.votepay_factor;
          auto to_per_vote_pay  = to_producers - to_per_block_pay;
```

Comment:
A zero continuous rate is allowed. The following issue and transfer actions should be wrapped in proper if statements, otherwise the action fails in the case of zero rate. Please add a corresponding test.